gulpfile.js

```diff
@@ -1,16 +1,37 @@
 'use strict';
 
+var child_process = require('child_process');
+
 var gulp = require('gulp');
-var beautify = require('gulp-jsbeautifier');
-var jshint = require('gulp-jshint');
-var mocha = require('gulp-mocha');
+
+var models = require('./models');
+var config = require('./lib/config');
 
 var js = ['**/*.js', '!node_modules/**/*.js'];
 var json = ['**/*.json', '!node_modules/**/*.json', '!tests/fixtures/**/*.json'];
 
+gulp.task('run', ['build'], function() {
+    // Create database tables (if needed) and start the server
+    models.sequelize.sync().then(function() {
```

Comment:
Should run [sequelize migrations](http://docs.sequelizejs.com/en/latest/docs/migrations/) as a separate step (and [as part of the heroku deployment](http://docs.sequelizejs.com/en/1.7.0/articles/heroku/#running-migrations)), rather than coupling a database sync to the web process. A sync on an existing database could mess things up, right?